### DISCUSSION TITLE
Clear Code

### DISCUSSION THREAD

**Discussion by michael-azogu** [fastify/fastify#3764] [General]
codebase could use some more descriptive names

**Comment by mcollina** [comment]
Can you make a few examples? What name is there/what would you expect?

**Reply by michael-azogu** [reply]
[lib/hooks:82](https://github.com/fastify/fastify/blob/main/lib/hooks.js)

i = 0
c = 0

I can't suggest names at this time because I don't know what they mean, I'd have to check how they are used. Maybe I could contribute by giving clear names to variables & what not, helps to reduce friction for future contributors.

**Reply by michael-azogu** [reply]
Maybe index & count

**Reply by mcollina** [reply]
Thanks! PRs for improving those cases are always welcomed!
